readme.md

```diff
@@ -223,7 +223,7 @@ On cancel responses collected so far are returned.
 
 Type: `Function`
 
-Preanswer questions by passing an object with answers to `prompts.override`.
+Answer questions by passing an object with answers to `prompts.override`.
```

Comment:
This is not what the author intended. Please think of the program itself... These are pre-filled answers and not the answers by the user.

Reply:
Thank you @koppor. Please revert this change.